packages/utilities/gel-foundations/CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 <!-- prettier-ignore -->
 | Version | Description |
 |---------|-------------|
+| 3.1.0 | [PR#701](https://github.com/bbc/psammead/pull/701) Update README `rem` doc link |
```

Comment:
Should probably be a `3.0.1` as we are not adding any backward compatible functionality to psammead. Its only documentation.